components/ShowNutritionalTable.tsx

```diff
@@ -48,6 +62,7 @@ export default function ShowNutritionalTable() {
     <>
       <Stack>
         <Nutriscore grade={nutriscore} />
+        <Nutriscore grade={modifiedNutriscore} /> 
```

Comment:
You could move the modification here with something like `{nutriscore !== undeined && <Nutriscore .../>}